processor/transformprocessor/README.md

```diff
@@ -14,44 +14,131 @@
 
 - [Config](#config)
+- [Writing OTTL](#writing-ottl)
 - [Grammar](#grammar)
-- [Contexts](#contexts)
 - [Supported functions](#supported-functions)
 - [Examples](#examples)
 - [Troubleshooting](#troubleshooting)
 - [Contributing](#contributing)
+- [Feature Gate](#feature-gate)
 
 ## Config
 
-The transform processor allows configuring multiple context statements for traces, metrics, and logs.
-The value of `context` specifies which [OTTL Context](#contexts) to use when interpreting the associated statements, and can be either explicitly set or [inferred](#context-inference).
-The global conditions and statement strings, which must be OTTL compatible, will be passed to OTTL and interpreted using the associated context.
-The condition string should contain a Where clause body without the `where` keyword at the beginning.
+The Transform Processor allows configuring OTTL statements for traces, metrics, and logs.
 
-Each context will be processed in the order specified.
-Within a context, each global condition is checked and if any evaluates to true, the statements are executed in order.
-If a context doesn't meet any of the conditions, then the associated statement will be skipped.
+> [!NOTE]
+> If you don't know how to write OTTL statements yet, first see [Writing OTTL](#writing-ottl).
+
+### General Config
+
+> [!NOTE]
+> If you don't know how to write OTTL statements yet, first see [Writing OTTL](#writing-ottl).
+
+```yaml
+transform:
+  error_mode: ignore
+  <trace|metric|log>_statements: []
+```
+
+Only certain OTTL Path prefixes can be used for each signal:
+
+| Signal            | Path Prefix Values                             |
+|-------------------|------------------------------------------------|
+| trace_statements  | `resource`, `scope`, `span`, and `spanevent`   |
+| metric_statements | `resource`, `scope`, `metric`, and `datapoint` |
+| log_statements    | `resource`, `scope`, and `log`                 |
 
-Each statement may have a Where clause that acts as an additional check for whether to execute the statement.
+This means, for example, that you cannot use the Path `span.attributes` within the `log_statements` configuration section.
 
-The transform processor also allows configuring an optional field, `error_mode`, which will determine how the processor reacts to errors that occur while processing a statement.
-The top-level `error_mode` can be overridden at context statements level, offering more granular control over error handling. 
+`error_mode`: determines how the processor reacts to errors that occur while processing a statement.
+If the top-level `error_mode` is not specified, `propagate` will be used.
+The top-level `error_mode` can be overridden at context statements level, offering more granular control over error handling. If context statement `error_mode` is not used, the top-level `error_mode` is used.
```

Comment:
```suggestion
The top-level `error_mode` can be overridden at statement group level, offering more granular control over error handling. If the statement group `error_mode` is not specified, the top-level `error_mode` is applied.
```
Maybe, to keep the naming similar to the new config name `Statement Group Config`?